ssh/DOCS.md

```diff
@@ -29,6 +29,13 @@ ha help
 
 You can access the web terminal by clicking the "Open Web UI" button on this add-on's Info tab. If you set the "Show in sidebar" setting (found on the same Info tab) to "on", a shortcut is added to the sidebar allowing you to access the web terminal quickly.
 
+To copy text from the Web UI:
+- hold down the SHIFT key
+- select the text you want to copy
+- when you release the left mouse button the text will copy to your system clipboard
+
+To paste text in the Web UI press SHIFT + INSERT
```

Comment:
```suggestion
You can copy text by holding down the shift key and then selecting the text to copy. Releasing the mouse button will copy the text to your system clipboard.
Use SHIFT + INSERT to paste text to the terminal.
```
Rewording suggestion.

Reply:
I think the step by step style is kinda nice. But I like some your suggestion style wise. Made a separate suggestion.